Fix NoClassDefFoundError ShutdownHookManager in tests

This does not affect the outcome of the test ... but can be misleading. I think this appeared with the latest `Beam` version.

```
Exception in thread "Thread-43" java.lang.NoClassDefFoundError: org/apache/hadoop/util/ShutdownHookManager$2
	at org.apache.hadoop.util.ShutdownHookManager.getShutdownHooksInOrder(ShutdownHookManager.java:124)
	at org.apache.hadoop.util.ShutdownHookManager$1.run(ShutdownHookManager.java:52)
Caused by: java.lang.ClassNotFoundException: org.apache.hadoop.util.ShutdownHookManager$2
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

--- a/.jvmopts
+++ b/.jvmopts
@@ -5,3 +5,7 @@
 -XX:ReservedCodeCacheSize=512m
 -XX:+TieredCompilation
 -XX:+UseParallelGC
+
+# https://github.com/spotify/scio/issues/2096
+# https://github.com/sbt/sbt/issues/4794#issuecomment-500602546
+-Dsbt.classloader.close=false

